src/screens/app/FeedScreen.js

```diff
@@ -15,7 +17,30 @@ export default class FeedScreen extends React.Component {
     this.state = {};
   }
 
+  componentDidMount() {
+    this.forbidFunction();
+  }
+
+  async forbidFunction() {
+    try {
+      const result = await NativeModules.PreventScreenshotModule.forbid();
+      console.log(result);
+    } catch (e) {
+      console.log(e);
+    }
+  }
+
+  async allowFunction() {
+    try {
+      const result = await NativeModules.PreventScreenshotModule.allow();
+      console.log(result);
+    } catch (e) {
+      console.log(e);
+    }
+  }
```

Comment:
Why are both of these functions being passed in, and where exactly will they be called? Is this for allowing specific users to decide which post can be screenshotted or not? Or are we allowing the whole app to be forbid or not forbid(but adding in notifications)